build.sbt

```diff
@@ -24,10 +24,13 @@ lazy val compile = Seq(
 lazy val scope: String = "test,it"
 
 lazy val test = Seq(
+  "com.github.tomakehurst" % "wiremock" % "1.58" % scope,
```

Comment:
we should try to use the latest stable version for this dependency

Reply:
Updated wiremock to latest version 2.19.0